zetaclient/chains/solana/observer/inbound_test.go

```diff
@@ -207,3 +208,51 @@ func Test_ParseInboundAsDeposit(t *testing.T) {
 		require.EqualValues(t, eventExpected, event)
 	})
 }
+
+func Test_ParseInboundAsDepositSPL(t *testing.T) {
```

Comment:
Consider splitting this test into 2 parts:
1. Parse raw SOL tx into "deposit spl" struct (inside solana package)
2. Invoke `ObserveInbound()` and "catch" a vote for inbound tx. Example: https://github.com/zeta-chain/node/blob/041df59fe4d012ecd293a0bb530e1c25dae9a381/zetaclient/chains/ton/observer/inbound_test.go#L120